dev/run_tests.py

```diff
@@ -113,6 +113,9 @@ def run_examples() -> List[str]:
         if os.path.isdir(example):
             if example.split("/")[-1].split("-")[0] in legacy_dirs:
                 continue
+            elif example.split("/")[-1] == "18-ad_afqmc":
```

Comment:
This is dangerous as things break all the time. Why is mpi4py required?

Reply:
An alternative would be to install mpi4py for the CI just so things are tested.

Reply:
Yes, please make sure this runs because the setup is non-trivial (for example, there is no input file present in the folder, is this generated during prep_ad?

Reply:
Oh there is a .json input file which is not generated by prep_ad

Reply:
The .h5 files are generated using prep_ad. I tested it on my local environment and it runs normally, but I don't know how to make it run in CI, should I use mpirun -np xx python xx?

Reply:
I can try to modify the example so that it can run without mpi. This would be better

Reply:
Don't add the files The example should generate ideally generate the files on the fly and run_afqmc should then run so it's more of an integration test.

Reply:
Oh ok, this is good. Thanks!